anyway/parsers/news_flash.py

```diff
@@ -29,11 +29,15 @@ def update_all_in_db(source=None, newsflash_id=None):
         newsflash_items = db.get_all_newsflash()
 
     for newsflash in newsflash_items:
-        classify = news_flash_classifiers[newsflash.source]
-        newsflash.organization = classify_organization(newsflash.source)
-        newsflash.accident = classify(newsflash.description or newsflash.title)
-        if newsflash.accident:
-            extract_geo_features(db, newsflash)
+        if update_cbs_location_only:
+            if newsflash.accident:
+                extract_geo_features(db, newsflash, update_cbs_location_only=True)
+        else:
+            classify = news_flash_classifiers[newsflash.source]
+            newsflash.organization = classify_organization(newsflash.source)
+            newsflash.accident = classify(newsflash.description or newsflash.title)
+            if newsflash.accident:
+                extract_geo_features(db=db, newsflash=newsflash, update_cbs_location_only=False)
```

Comment:
Why not reverse the if condition, and avoid having the inner if twice:
```
        if not update_cbs_location_only:
            classify = news_flash_classifiers[newsflash.source]
            newsflash.organization = classify_organization(newsflash.source)
            newsflash.accident = classify(newsflash.description or newsflash.title)
         if newsflash.accident:
            extract_geo_features(db=db, newsflash=newsflash, update_cbs_location_only)
```

Reply:
Fixed, thanks